src/rollbar.ts

```diff
@@ -1,114 +1,27 @@
 /**
  * Wrapper around Rollbar, for error reporting.
- */ /** */
-
-import * as path from 'path';
-import * as vscode from 'vscode';
-
-import Rollbar = require('rollbar');
+ */
```

Comment:
Do you plan to rename the rollbar in a subsequent PR?

Reply:
I didn't have such plan. I can do it in this PR if you think renaming is necessary.

Reply:
Don't do it in this PR since the diff noise will be too large